images/win/scripts/Installers/Update-AndroidSDK.ps1

```diff
@@ -100,7 +101,9 @@ Push-Location -Path $sdk.FullName
     "ndk-bundle"
 
     # Android NDK root path.
-    $ndk_root = "C:\Program Files (x86)\Android\android-sdk\ndk-bundle"
+    $ndk_root = "$sdk_root\ndk-bundle"
+    # Create symlink for backward compatibility
+    mklink /J "C:\Program Files (x86)\Android\android-sdk" $sdk_root
```

Comment:
Could we use environment variable `$env:ProgramFiles(x86)`, instead of hardcoded path?

Reply:
Thank you, added.